public/js/post_action.js

```diff
@@ -26,8 +26,33 @@ jQuery(function ($) {
     }
   }
 
+  function scroll_action_card_scroll() {
+    const $boxoutScroll = $('.post-content').find('> #action-card-scroll').not('.hidden');
+    const postStart = $('.post-content').position().top;
+    const postHeight = $('.post-content').outerHeight();
+    if ($boxoutScroll.length > 0) {
+      if ($(window).scrollTop() > postStart && $(window).scrollTop() < (postHeight - postStart)) {
```

Comment:
On a small screen (around 500px) with a short post of just a paragraph (more or less the size of `postStart`), the card never shows up. I don't know how it should behave in this condition.

Reply:
Yes that's a good point, already noticed by Will and he wants to discuss it with Magali!